code/modules/admin/verbs/mapping.dm

```diff
@@ -30,6 +30,12 @@ var/intercom_range_display_status = 0
 		src.pixel_x = -224
 		src.pixel_y = -224
 
+/obj/effect/debugging/mapfix_marker

```

Comment:
It shows up in the new map if they modify the same tile, and is evidence that the map was not correctly de-conflicted.